api/v1beta1/aerospikecluster_validating_webhook.go

```diff
@@ -1037,6 +1037,8 @@ func validateNsConfUpdate(
 
 	newNsConfList := newConf["namespaces"].([]interface{})
 
+	deviceList := map[string]string{}
```

Comment:
you should comment about purpose of this data structure and indicate what represent key and value.

Reply:
Done